packages/@magic-sdk/types/src/modules/user-types.ts

```diff
@@ -120,19 +120,22 @@ export enum RecoverAccountEventOnReceived {
   LoginThrottled = 'login-throttled',
   InvalidSmsOtp = 'invalid-sms-otp',
   SmsVerified = 'sms-verified',
+  AccountRecovered = 'account-recovered',
```

Comment:
We need one more event to indicate the next step, e.g. 'update-email-required'

Reply:
80795e61b3d4cefd344f4d2e26b29e4fd084639b